metaflow/plugins/aip/aip.py

```diff
@@ -901,26 +934,10 @@ def _set_container_labels(self, container_op: ContainerOp):
                 "tags.ledger.zgtools.net/ai-experiment-name", self.experiment
             )
 
-        # - In context of Zillow CICD self.username == "cicd_compile"
-        # - In the context of a Zillow NB self.username == METAFLOW_USER (user_alias)
-        # - In the context of Metaflow integration tests self.username == USER=$GITLAB_USER_EMAIL
-        owner = self.username
-        if "@" in owner:
-            owner = owner.split("@")[0]
-        container_op.add_pod_label("zodiac.zillowgroup.net/owner", owner)
-
-        # Add in Zodiac service and team labels to the aip pods if the environment variable is
-        # present in the notebook (individual profile notebooks only) and set them. These labels
-        # are not being added by poddefaults as they were removed. Workflows launched in project
-        # profiles still get these labels added via poddefaults. Also adds in logging topic
-        # annotation as this value is specific to zodiac service as well.
         if ZILLOW_ZODIAC_SERVICE and ZILLOW_ZODIAC_TEAM:
-            container_op.add_pod_label(
-                "zodiac.zillowgroup.net/service", ZILLOW_ZODIAC_SERVICE
-            )
-            container_op.add_pod_label(
-                "zodiac.zillowgroup.net/team", ZILLOW_ZODIAC_TEAM
-            )
+            # Add a logging topic annotation specific to the Zodiac service.
+            # This is done to support user-supplied Zodiac service per AIP Notebook.
+            # Please see comments on how and why ZILLOW_ZODIAC_SERVICE label for more.
             container_op.add_pod_annotation(
                 "logging.zgtools.net/topic",
                 f"log.fluentd-z1.{ZILLOW_ZODIAC_SERVICE}.dev",
```

Comment:
Not really part of this PR, but I'm curious why logging topic always ends with `.dev`. Is there any chance it can also be `.stage` / `.prod` etc?

Reply:
Nice question!:
* The intent of this code is only for Notebook launched workflows and hence dev is ok.
* However, I noticed that `logging.zgtools.net/topic` is deprecated for `logging.zgtools.net/index`,
  * see webhook code https://gitlab.zgtools.net/analytics/artificial-intelligence/ai-platform/aip-infrastructure/aip-admission-webhook-service/-/blob/main/aip_admission_webhook_service/components/pod/config_utils.py#L110
  * @aaron-arellano , do you believe this should be changed to use `logging.zgtools.net/index` ?
      * or is this a question for Max, as part of the ZAP migration? see how `logging.zgtools.net/topic` is specified by customers in `kubeflow-k8s-profile`s and not applied with the webhook: https://gitlab.zgtools.net/analytics/artificial-intelligence/ai-platform/aip-infrastructure/kubeflow-k8s-profiles/-/blob/main/profiles/prod/rental-grs-prod/rental-grs-prod.yaml#L9

Reply:
In ZAP, we will change these customized labels from namespace to AIPEnvironment which is ZAP specific resource in namespace level. We will also update these config_util from AIPEnvironment annotations instead of namespace. But for this edge case in rental_grs in profile about whether to use index or topic, I'll leave it to the original owner for this logic update as I'm not sure it will impact the existing rental-grs logging if we update it to` logging.zgtools.net/index`.

Reply:
note: updated the annotation to `"logging.zgtools.net/index",`